channeldb/payments.go

```diff
@@ -1297,6 +1303,13 @@ func deserializeHop(r io.Reader) (*route.Hop, error) {
 		h.Metadata = metadata
 	}
 
+	attributableErrorType := uint64(record.AttributableErrorOnionType)
+	if _, ok := tlvMap[attributableErrorType]; ok {
+		delete(tlvMap, attributableErrorType)
```

Comment:
Because right below here, the remaining records are stored as custom records. If we wouldn't delete here, the attr error record would also show up as a custom record.